src/main/resources/static/js/hposummary.js

```diff
@@ -0,0 +1,65 @@
+// Javascript for the hposummary table on phenotypes view
+
+function showLabs(d) {
+	// `d` is the original data object for the row
+	let table = "<div class='card float-left'>"
+			+ "<table class='table-bordered sub-table'>"
+			+ "<thead>"
+			+ "<tr class='table-warning'><th>FHIR Id</th><th>LOINC</th><th>Description</th><th>Value</th><th>Date</th></tr>"
+			+ "</thead>" + "<tbody>";
+	for (i = 0; i < d.observations.length; i++) {
+		let observation = d.observations[i];
+		table += "<tr>" + "<td>" + observation.fhirId + "</td>" + "<td>"
+				+ observation.loincId + "</td>" + "<td>"
+				+ observation.description + "</td>" + "<td>"
+				+ observation.value + "</td>" + "<td>" + observation.date
+				+ "</td>" + "</tr>";
+	}
+	table += "</tbody></table></div>";
+	return table;
+}
+
+let patientId = $("#patient_id").html();
+
```

Comment:
When this file is included in a page anything defined in it is available globally. To avoid potential conflicts with other libraries, you should probably include the patientId variable within the document.ready. You could also put the showLabs function there.